app/src/main/java/com/eveningoutpost/dexdrip/Models/UserError.java

```diff
@@ -142,13 +142,6 @@ public synchronized static void cleanupRaw() {
     }
 
 
-    public static List<UserError> all() {
```

Comment:
any particular reason to remove this? proguard will strip unused methods from the apk automatically.

Reply:
Well, I thought that code is easier to read if there are no unused functions.
I Can return this if you think it has value.

Reply:
Please keep it for now.